server/modules/nodeDef/manager/nodeDefManager.js

```diff
@@ -170,14 +177,16 @@ export const updateNodeDefProps = async (
       ...(R.isEmpty(propsAdvanced) ? {} : { propsAdvanced }),
     }
 
-    const [nodeDef] = await Promise.all([
-      NodeDefRepository.updateNodeDefProps(surveyId, nodeDefUuid, props, propsAdvanced, t),
+    const nodeDef = await NodeDefRepository.fetchNodeDefByUuid(surveyId, nodeDefUuid, true, false, t)
+
+    const [nodeDefUpdated] = await Promise.all([
+      NodeDefRepository.updateNodeDefProps(surveyId, nodeDefUuid, props, propsAdvanced, NodeDef.isAnalysis(nodeDef), t),
```

Comment:
i would conditionally update the column (see https://dba.stackexchange.com/questions/39815/use-case-to-select-columns-in-update-query)